### DISCUSSION TITLE
[Question] which version of server supported?

### DISCUSSION THREAD

**Discussion by anatawa12** [sladkoff/minecraft-prometheus-exporter#118] [Q&A] [answered]
which version of the Bukkit minecraft server is the target version(s) of this pluign? 
Are there any (old) versions that support(ed) 1.7.10?

**Comment by HSoeren** [comment, marked as answer]
I'm not the author, just user. According to the plugin page over at spigot mc all versions from 1.11 to 1.16 are currently supported. For other versions personally I would try and see if its working or not.
